README.md

```diff
@@ -48,7 +48,7 @@ Description:
   Valet is a tool to help plan and facilitate migrations to GitHub Actions.
 
 Usage:
-  gh-valet [command] [options]
+  gh valet [command] [options]
```

Comment:
I don't believe modifying the readme is enough 

You can test it yourself with
```bash
dotnet build src/valet.sln
```
Followed by
```bash
dotnet run --project src/Valet/Valet.csproj -- --help
```

Reply:
Ya, was just talking to @ethanis about that, potentially could customize it with something like [this](https://docs.microsoft.com/en-us/dotnet/standard/commandline/customize-help)